Artifacts/linux-secure-mariadb/README.md

```diff
@@ -0,0 +1,21 @@
+# Linux Secure MariaDB Artifact
+This Azure DevTest artifact allows the user to secure MariaDB installations already present on a Azure DevTest Lab VM (for example installed via Apt/Yum Package Manager Artifact). This artifact applies to any Linux distribution.
+
+The script is based on [Twitters implementation of mysql_secure_installation.sh](https://github.com/twitter/mysql/blob/master/scripts/mysql_secure_installation.sh) and basically does the following:
+1. Set root password and make sure that nobody can access the server without a password
+2. Remove anonymous users
+3. Disallow root login remotely
+4. Remove test database and access to it
+5. Reload privilege tables
+
+## Usage 
+The script is intended for use in the Azure DevTest Labs artifact system, and the parameters for the artifact are fed directly
+into the script. However, you can run it from any bash shell using the following format:
+
+        bash> ./linux-secure-mariadb.sh [-p=password]  
```

Comment:
This bash example doesn't match the syntax we're using in the `artifactfile.json` file. Can we in fact use both parameter formats (i.e. `-p=password` and `-p password`)?